src/payment/strategies/ppsdk/ppsdk-payments-api-response.ts

```diff
@@ -0,0 +1,3 @@
+import { Response } from '@bigcommerce/request-sender';
+
+export type PaymentsAPIResponse = Response<unknown>;
```

Comment:
Similar to `initializationStrategy` - this API response will eventually be a large union of various responses. To defend against this `unknown` is being used to force defensive type guarding prior to any  real use.

Reply:
Similarly to https://github.com/bigcommerce/checkout-sdk-js/pull/1150/files#r649623606, we don't know what the final union or competent shapes will be. 

`unknown` seemed safest, forcing the use of type guards later in the consuming code.